tests/baselines/reference/tsserver/projectReferences/project-is-indirectly-referenced-by-solution.js

```diff
@@ -974,14 +966,9 @@ Project '/user/username/projects/myproject/tsconfig-src.json' (Configured)
 -----------------------------------------------
 event:
     {"seq":0,"type":"event","event":"projectLoadingFinish","body":{"projectName":"/user/username/projects/myproject/tsconfig-src.json"}}
-Search path: /user/username/projects/myproject/src
-For info: /user/username/projects/myproject/src/main.ts :: Config file name: /user/username/projects/myproject/tsconfig.json
-Search path: /user/username/projects/myproject/src
-For info: /user/username/projects/myproject/src/main.ts :: Config file name: /user/username/projects/myproject/tsconfig.json
-Search path: /user/username/projects/myproject/src/helpers
-For info: /user/username/projects/myproject/src/helpers/functions.ts :: Config file name: /user/username/projects/myproject/tsconfig.json
 Search path: /user/username/projects/myproject/src/helpers
 For info: /user/username/projects/myproject/src/helpers/functions.ts :: Config file name: /user/username/projects/myproject/tsconfig.json
+Finding references to /user/username/projects/myproject/src/helpers/functions.ts position 13 in project /user/username/projects/myproject/tsconfig-src.json

```

Comment:
In a few places, a search happens earlier than it used to.  This is part of trying to eliminate duplicate searches (pulling the high-quality search ahead of the low-quality search so the low-quality search can be skipped).